GitTfs.Vs2015/paket.references

```diff
@@ -1,6 +1,2 @@
-//If you plan to update this file, you must first absolutly read our documentation on the subject
-//because there is some subtleties on how we use it in our project:
-//https://github.com/git-tfs/git-tfs/blob/master/doc/paket.md
```

Comment:
I understand better the problem. In fact, we can't add comments in the file. I didn't see well but from the start, we are talking about `paket.references` and I thought we were talking about `paket.dependencies`.

Indeed, I had added the comments in the 2 files (things I didn't remember!). The 2nd one without testing :(

There is no real reason to let this comment in this file so I am ok to remove it and to merge the PR.

All is OK!